src/IISIntegration/src/Microsoft.AspNetCore.Server.IISIntegration/Microsoft.AspNetCore.Server.IISIntegration.csproj

```diff
@@ -9,6 +9,11 @@
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
   </PropertyGroup>
 
+  <ItemGroup>
+    <!-- TODO use SharedSourceRoot instead of hard coded path. Issues with including Directory.Build.props -->
+    <Compile Include="$(RepositoryRoot)..\Shared\HttpSys\**\*.cs" />
```

Comment:
This isn't needed in IISIntegration.

Reply:
Weird, don't know why this ended up in here. I'll remove